### ISSUE TITLE
Phantom stops at the end of each scene.

### ISSUE THREAD

**Issue by Hediby** [lyft/l5kit#235]
Hi ! I noticed that when using the class `EgoDataset`,  the `target_position` for frames at the end of a scene is always a stop, even when it shouldn't be. You can visualize these 'phantom' stops by watching two consecutive scenes in the dataset, but you can also do this:

First load the dataset:
```
os.environ["L5KIT_DATA_FOLDER"] = "/path/to/your/dataset"
dm = LocalDataManager(None)
cfg = load_config_data("./config.yaml")

rasterizer = build_rasterizer(cfg, dm)
train_zarr = ChunkedDataset(dm.require(cfg["train_data_loader"]["key"])).open()
train_dataset = EgoDataset(cfg, train_zarr, rasterizer)
```

Then, we can take the last frame of a scene `scene_idx`, and the first frame of the next scene `scene_idx + 1`:

```
scene_idx = 15 # I took this as an example, but the same behavior can be observed with any consecutive scenes
idx1 = train_dataset.get_scene_indices(scene_idx)[-1]
idx2 = train_dataset.get_scene_indices(scene_idx+1)[0]
data1 = train_dataset[idx1]
data2 = train_dataset[idx2]
```
We check that the two frames are consecutive:
```
delta_t = (data2['timestamp']-data1['timestamp'])*1e-10
print("%1.2f seconds between the two frames" % delta_t)
>>  0.02 seconds between the two frames
``` 
And then we look at their `target_positions`
```
print(data1['target_positions'][:,0])
print(data2['target_positions'][:,0])
>>
[0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.
 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
[ 0.8103376  1.609011   2.4134367  3.214226   4.000056   4.7925463
  5.580153   6.366665   7.157464   7.944052   8.724877   9.497141
 10.270997  11.045094  11.81986   12.593572  13.355322  14.109047
 14.830419  15.578126  16.331459  17.071692  17.820902  18.567629
 19.316858  20.065075  20.810734  21.55106   22.291332  23.028435
 23.767242  24.495708  25.219261  25.94316   26.65681   27.37983
 28.090437  28.794268  29.509027  30.214304 ]
```

Is it the expected behavior for this class?

**Comment by Hediby** [comment]
If we visualize the two consecutive frames, we have this:

![image](https://user-images.githubusercontent.com/12557936/107017035-f0f4c980-679e-11eb-8008-5e5bbe3a979d.png)

**Comment by lucabergamini** [comment]
This is expected. We always reason on a single scene only, so when you're at the end of it there is no future to be used. This is an internal design choice. However, we provide availability for future steps so you can remove them from the loss computation and not penalise the model from those ghost stops
